bin/import_standards_from_csv

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+require_relative '../dashboard/config/environment'
+
+# Loads/merges the data from a CSV into the Standards table.
+# Can be used to overwrite the description and concept of
+# existing Standards and to create new Standards.
+# Will not delete existing Standards.
+
+# @param csv_path [String] The path to the CSV file.
+# Expected columns:
+# - organization
+# - organization_id
+# - description
+# - concept
+def main(csv_path)
+  created = 0
+  updated = 0
+  CSV.foreach(csv_path, {headers: true, quote_char: "\x00"}) do |row|
+    parsed = {
+      organization: row['organization'],
+      organization_id: row['organization_id'],
+      description: row['description'],
+      concept: row['concept']
+    }
+    loaded = Standard.find_by({organization: parsed[:organization], organization_id: parsed[:organization_id]})
+    if loaded.nil?
+      begin
+        if Standard.new(parsed).save!
+          created += 1
+        end
+      rescue => error
+        puts "Error when processing #{parsed[:organization]} #{parsed[:organization_id]}: #{error.message}"
+      end
+    else
+      loaded.assign_attributes(parsed)
+      if loaded.changed?
+        if loaded.update!(parsed)
```

Comment:
it would be more clear to just use `save!` if you are not passing in any arguments to `update!`.  `update!` will also raise (not return false) if the thing it is saving is invalid, so you don't need to check the return value here either -- but you may want to handle the exception like you did when you rescued the error on creation.